openedx/core/djangolib/oauth2_retirement_utils.py

```diff
@@ -0,0 +1,41 @@
+from oauth2_provider.models import (
+    AccessToken as DOTAccessToken,
+    Application as DOTApplication,
+    Grant as DOTGrant,
+    RefreshToken as DOTRefreshToken,
+)
+from provider.oauth2.models import (
+    AccessToken as DOPAccessToken,
+    RefreshToken as DOPRefreshToken,
+    Grant as DOPGrant,
+)
+
+def delete_user_from(model, user_id):
+    user_query_results = model.objects.filter(user_id=user_id)
+
+    if not user_query_results.exists():
+        return False
+
+    user_query_results.delete()
+    return True
+
+def delete_from_oauth2_provider_accesstoken(user):
```

Comment:
Why not create a static class, but instead implementing all of these as utility functions?

Reply:
To clarify, I would wrap these methods in a class (let's say `DOTRetirementUtilities`) and make the methods static?